src/form/Form.jsx

```diff
@@ -1394,17 +1386,17 @@ function Form({
                             return (
                                 <Elements.TextField
                                     {...fieldProps}
-                                    lazy={false}
-                                    unmask
+                                    // lazy={false}
```

Comment:
don't leave commented out code in production

Reply:
Done